scripts/xtensa-build-all.sh

```diff
@@ -248,7 +253,12 @@ do
 		-DCMAKE_VERBOSE_MAKEFILE=ON \
 		..
 
-	make ${PLATFORM}_defconfig
+	if [[ "x$MAKE_DEFCONFIG" == "xyes" ]]
```

Comment:
Variable name may be misleading, I think `MAKE_MENUCONFIG` would be better name.

Reply:
@jajanusz fixed

Reply:
@ranj063 Thanks